### PR TITLE
Issue #3005411: Default users for testing

### PR DESCRIPTION
**Pull request by mtodor**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/BurdaMagazinOrg/thunder-distribution/blob/develop/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: modules/thunder_demo/content/user/09ec113d-32a1-4dcf-80d9-fc78069ce7b1.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        }
+    ],
+    "preferred_admin_langcode": [
+        {
+            "value": "en"
+        }
+    ],
+    "name": [
+        {
+            "value": "kayne"
```
Comment (dbosen) [inline modules/thunder_demo/content/user/09ec113d-32a1-4dcf-80d9-fc78069ce7b1.json]: I guess this should be Kanye :) - Well, actually it is just Ye now (https://en.wikipedia.org/wiki/Kanye_West#Name_%22Ye%22).

File: modules/thunder_demo/content/user/09ec113d-32a1-4dcf-80d9-fc78069ce7b1.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        {
+            "value": true,
+            "lang": "en"
+        }
+    ],
+    "metatag": [
+        {
+            "value": {
+                "title": "kayne | Site-Install",
+                "canonical_url": "http:\/\/default\/user\/4",
```
Comment (dbosen) [inline modules/thunder_demo/content/user/09ec113d-32a1-4dcf-80d9-fc78069ce7b1.json]: This is usually wrong, can we skip the metatag information? The node exports just contain the field_meta_tags values.

File: modules/thunder_demo/content/user/09ec113d-32a1-4dcf-80d9-fc78069ce7b1.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    ],
+    "roles": [
+        {
+            "target_id": "administrator"
+        }
+    ],
+    "default_langcode": [
+        {
+            "value": true,
+            "lang": "en"
```
Comment (dbosen) [inline modules/thunder_demo/content/user/09ec113d-32a1-4dcf-80d9-fc78069ce7b1.json]: Lang codes have to be removed everywhere, see node export.

**Review by dbosen — CHANGES_REQUESTED** [review]
I am not sure about this. I did not know, that so many user should be generated, with all the funny names :).
In the distribution we should just have one user per role with simple technical names. Editor, restricted-editor, seo. 
The more complex setups like this should be in the testing only module.

File: modules/thunder_demo/content/user/09ec113d-32a1-4dcf-80d9-fc78069ce7b1.json (hunk 1 above)
Comment (mtodor) [reply modules/thunder_demo/content/user/09ec113d-32a1-4dcf-80d9-fc78069ce7b1.json]: Ticket is defined on 24.09.2018. It as legacy naming. 🤣

**Comment by mtodor** [comment]
@dbosen I have addressed all the comments from the review. I've started work on a custom module with all listed testing users (btw. we need repo for that). And here we have one user per provided role. I would keep prefix `test-` before role to avoid possible conflicts with created `admin` or `administrator` user during installation process.

**Comment by chrfritsch** [comment]
Could we open the context of this PR a bit and also add some tags, that are connected to our articles?

That would be helpful for normal testing and also to see some differences on https://github.com/BurdaMagazinOrg/theme-thunder-admin/pull/166

**Comment by mtodor** [comment]
@chrfritsch - I would create new issue for adding of Tags for articles and make it in another PR. It's better to keep scopes smaller.

**Review by chrfritsch — DISMISSED** [review]
Please, then make it so.

We also need a follow-up to set the new users as authors of our nodes.

**Comment by mtodor** [comment]
Added issue for tags: https://www.drupal.org/project/thunder/issues/3008589
And for users assignment: https://www.drupal.org/project/thunder/issues/3008594

**Review by dbosen — APPROVED** [review]

**Pull request merged**